src/shared/schema/mix/actions/dash.ts

```diff
@@ -41,49 +41,76 @@ export const dashActions = {
         //   {entryId: "0epkfeanqv", type: "graph_wizard_node"}
         // * node script
         //   {entryId: "ilslg0le88", type: "graph_node"}
-        const result = await Promise.all(
-            entries.map(async (entry) => {
-                const {entryId, type, meta, links} = entry;
-                const {dataset} = links || {};
-                // deprecated
-                const {datasetId: metaDatasetId} = meta || {};
-                const datasetId = (dataset || metaDatasetId) as string | undefined;
-                const widgetType = type.match(/^[^_]*/)?.[0] || null;
-                // datasetsIds is now ignored because you need to find out the name for all datasets
-                // if (datasetsIds.includes(datasetId)) {
-                //     return {entryId, type: widgetType, datasetId};
-                // }
-                if (datasetId) {
-                    try {
-                        const {
-                            key,
-                            dataset: {result_schema},
-                        } = await typedApi.bi.getDatasetByVersion({
-                            datasetId,
-                            version: 'draft',
-                        });
+        const fetchEntries = entries.map(async (entry) => {
+            const {entryId, type, meta, links} = entry;
+            const {dataset} = links || {};
+            // deprecated
+            const {datasetId: metaDatasetId} = meta || {};
+            const datasetId = (dataset || metaDatasetId) as string | undefined;
+            const widgetType = type.match(/^[^_]*/)?.[0] || null;
```

Comment:
Maybe `at least one` regex with `+`  sign
![image](https://github.com/datalens-tech/datalens-ui/assets/5037509/7382ecc8-bbd6-4874-a2ef-ecc3b2053876)

Reply:
If you check full expression, you will see no changes in result:
<img width="222" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/98329176/f6b698c6-6918-4fce-85cf-18619b9e58ad">